OWNERS

```diff
@@ -1,15 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
 approvers:
-- jsafrane
-- lpabon
-- msau42
-- saad-ali
-- xing-yang
+- kubernetes-csi-approvers
+
 reviewers:
-- chrishenzie
+- kubernetes-csi-reviewers
```

Comment:
Move davidz627, lpabon, sbezverk to emeritus_reviewers

Reply:
Listing davidz627 here would make sense (not listed under the shared file), but for lpabon and sbezverk it would be duplication.

Reply:
I agree.